api/tests/e2e/api/test_inclusion_data.py

```diff
@@ -663,6 +663,33 @@ def test_list_services_by_types(api_client, url):
     assert_paginated_response_data(response.json(), total=0)
 
 
+@pytest.mark.with_token
+@pytest.mark.parametrize("url", ["/api/v0/services", "/api/v0/search/services"])
+def test_list_services_by_score_qualite(api_client, url):
+    service_1 = factories.ServiceFactory(score_qualite=0.5)
+    service_2 = factories.ServiceFactory(score_qualite=0.7)
+    factories.ServiceFactory(score_qualite=0.2)
+
+    response = api_client.get(
+        url,
+        params={"score_qualite_minimum": 0.5},
```

Comment:
:+1: nickel d'avoir mis ce edge case

peut-être :

```suggestion
        params={"score_qualite_minimum": service_1.score_qualite},
```

pour bien voir ce que la valeur n'est pas choisie au hasard ?